opm/simulators/wells/PerforationData.hpp

```diff
@@ -0,0 +1,36 @@
+/*
+  Copyright 2019 SINTEF Digital, Mathematics and Cybernetics.
+
+  This file is part of the Open Porous Media project (OPM).
+
+  OPM is free software: you can redistribute it and/or modify
+  it under the terms of the GNU General Public License as published by
+  the Free Software Foundation, either version 3 of the License, or
+  (at your option) any later version.
+
+  OPM is distributed in the hope that it will be useful,
+  but WITHOUT ANY WARRANTY; without even the implied warranty of
+  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+  GNU General Public License for more details.
+
+  You should have received a copy of the GNU General Public License
+  along with OPM.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#ifndef OPM_PERFORATIONDATA_HEADER_INCLUDED
+#define OPM_PERFORATIONDATA_HEADER_INCLUDED
+
+namespace Opm
+{
+
+/// Static data associated with a well perforation.
+struct PerforationData
+{
```

Comment:
The parser uses connection for perforation. The well model a combination. Should we try to synchronize?

Reply:
A good idea. ConnectionData is a better name. There are many uses of the term perforation in the code though (and not from this PR), in variable names for example. I suggest we go on a "perforation hunt" after this PR. Is there any use for the concept *perforation* at all, or are *connection* and *completion* the concepts we need?